idea_town/frontend/static-src/app/templates/account-inactive-page.js

```diff
@@ -1,11 +1,16 @@
 export default `
-  <div class="page">
-    <section class="page main-content" data-hook="account-inactive-page">
-      <header data-hook="main-header"></header>
-      <h1>Account Inactive</h1>
-      <h2>We're not quite ready for you yet!<br>
-          Please come back in a few weeks.</h2>
+  <div class="page" id="full-page-notice">
+    <section class="main-content" data-hook="account-inactive-page">
+      <div class="vertical-centering-container">
+        <div id="beta-notice-modal" class="delayed-fade-in">
+          <h1>Thanks for signing up!</h1>
+          <h2>We'll let you know as soon as Idea Town is ready.</h2>
```

Comment:
We collect the email address passed along with the FxA sign-in, introduced here:

https://github.com/mozilla/idea-town/issues/128#issuecomment-148826004
https://github.com/mozilla/idea-town/commit/f1d09e9cb35d3c1e5b0df39aa31249d13bd61074